File: src/main/java/org/smarthata/service/tm/command/MessagesCommand.java
package org.smarthata.service.tm.command;

import org.apache.logging.log4j.util.Strings;
import org.springframework.stereotype.Service;
import org.telegram.telegrambots.meta.api.methods.BotApiMethod;


@Service
public class MessagesCommand extends AbstractCommand {

    private static final String MESSAGES = "messages";

    public MessagesCommand() {
        super(MESSAGES);
    }

    @Override
    public BotApiMethod<?> answer(CommandRequest request) {
        String text = Strings.join(request.getPath().iterator(), ',');
        return aSimpleSendMessage(request.getChatId(), text).build();
    }
}

File: src/main/kotlin/org/smarthata/service/tm/command/MessagesCommand.kt
package org.smarthata.service.tm.command

import org.springframework.stereotype.Service
import org.telegram.telegrambots.meta.api.methods.BotApiMethod

@Service
class MessagesCommand : AbstractCommand(MESSAGES) {
    override fun answer(request: CommandRequest): BotApiMethod<*> {
        val text = request.path.joinToString(",")
        return createTmMessage(request, text)
    }

    companion object {
        private const val MESSAGES = "messages"
    }
}
